File: backend/src/main/java/com/example/demo/dto/ProductExitRequest.java
package com.example.demo.dto;

/**
 * Clase ProductExitRequest del sistema de inventario.
 */
public class ProductExitRequest {

    private String product;
    private Integer quantity;

    public String getProduct() {
        return product;
    }

    public void setProduct(String product) {
        this.product = product;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public void setQuantity(Integer quantity) {
        this.quantity = quantity;
    }
}

File: AppAndroid/TFGApp/app/src/main/java/com/example/tfgapp/data/model/ProductExitRequest.kt
// Archivo comentado: app/src/main/java/com/example/tfgapp/data/model/ProductExitRequest.kt
// Explica la responsabilidad de esta clase y los pasos principales del flujo.

package com.example.tfgapp.data.model


// Modelo de datos usado para enviar o recibir información de la API.
data class ProductExitRequest(
    val product: String,
    val quantity: Int
)
